cmd/control/config.go

```diff
@@ -290,13 +291,48 @@ func validate(c *cli.Context) error {
 func inputBytes(c *cli.Context) ([]byte, error) {
 	input := os.Stdin
 	inputFile := c.String("input")
-	if inputFile != "" {
-		var err error
-		input, err = os.Open(inputFile)
-		if err != nil {
-			return nil, err
-		}
-		defer input.Close()
+	if inputFile == "" {
+		return nil, errors.New("input parameter can not be empty")
+	}
+	if hasCRLF(inputFile) {
+		return []byte{}, errors.New("file format shouldn't contain CRLF characters")
+	}
+	input, err := os.Open(inputFile)
+	if err != nil {
+		return nil, err
 	}
+	defer input.Close()
 	return ioutil.ReadAll(input)
 }
+
+func hasCRLF(fileName string) bool {
```

Comment:
Hard to read, can we make it simple?

Not sure if this is fine:
```
content, err := ioutil.ReadFile(filename)
return bytes.Contains(content, []byte{13, 10})
```

refer to: https://www.programming-books.io/essential/go/1d3abcf6f17c4186bb9617fa14074e48-normalize-newlines